### PR TITLE
dep(solang-parser): 0.1.17

### PR DESCRIPTION
**Pull request by rkrasiuk**
## Motivation

new release of solang-parser `0.1.17`

## Solution

refine doc comment handling

depends on [ethers-rs#1548](https://github.com/gakonst/ethers-rs/pull/1548)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -55,10 +55,10 @@ panic = "abort"
...
-#ethers-signers = { path = "../ethers-rs/ethers-signers" }
-#ethers-etherscan = { path = "../ethers-rs/ethers-etherscan" }
-#ethers-solc = { path = "../ethers-rs/ethers-solc" }
+# [patch."https://github.com/gakonst/ethers-rs"]
+# ethers = { path = "../ethers-rs" }
+# ethers-core = { path = "../ethers-rs/ethers-core" }
+# ethers-providers = { path = "../ethers-rs/ethers-providers" }
+# ethers-signers = { path = "../ethers-rs/ethers-signers" }
+# ethers-etherscan = { path = "../ethers-rs/ethers-etherscan" }
+# ethers-solc = { path = "../ethers-rs/ethers-solc" }
```
Comment (mattsse) [inline Cargo.toml:64]: revert

**Review by mattsse — APPROVED** [review]
lgtm, pending ethers bump

**Comment by gakonst** [comment]
bumped

**Review by gakonst — APPROVED** [review]
lgtm pending ethers bump

**Comment by gakonst** [comment]
something failing hehe

**Pull request merged**